tests/eval/stdin-tty/input.joke

```diff
@@ -0,0 +1,12 @@
+(ns stdin-tty-test
```

Comment:
This seems to duplicate `test-exec` above (using a different method of testing). I think it makes sense to use `deftest` method when possible, so this test can be removed.